### PR TITLE
Add initial code base

### PR DESCRIPTION
**Pull request by lojzatran**
- Set folder structure
- Add example files
- Set linters
- Add CTP client
- Add logger
- Add rc config
- Add docker
- Add renovate file

### CODE REVIEW

File: .subscriptionsrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (ahalberkamp) [inline .subscriptionsrc:1]: I would rename it to ctpCredentials

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # commercetools-subscriptions
+
 Cron job based service which manages subscriptions, reminders and triggers subscription orders based on the schedule
```
Comment (ahalberkamp) [inline README.md:3]:
```suggestion
Cron job based service which manages subscriptions, reminders and triggers subscription orders based on a schedule
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "commercetools-subscriptions",
+  "version": "0.0.1",
+  "description": "Cron job based service which manages subscriptions, reminders and triggers subscription orders based on the schedule",
```
Comment (ahalberkamp) [inline package.json:4]:
```suggestion
  "description": "Cron job based service which manages subscriptions, reminders and triggers subscription orders based on a schedule",
```

File: renovate.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": ["config:base", ":disableDependencyDashboard"],
+  "baseBranches": ["master"],
```
Comment (ahalberkamp) [inline renovate.json]:
```suggestion
  "baseBranches": ["main"],
```

File: .subscriptionsrc (hunk 1 above)
Comment (lojzatran) [reply .subscriptionsrc:1]: For now it is only the credentials, but with time there will be more configs.

File: README.md (hunk 2 above)
Comment (lojzatran) [reply README.md:3]: I guess here it doesn't matter. I'm not sure if your suggestion is correct, so I'll leave it as it is.

File: package.json (hunk 3 above)
Comment (lojzatran) [reply package.json:4]: https://github.com/commercetools/commercetools-subscriptions/pull/1#discussion_r805729704

File: .subscriptionsrc (hunk 1 above)
Comment (ahalberkamp) [reply .subscriptionsrc:1]: ok

**Review by ahalberkamp — APPROVED** [review]

**Pull request merged**